Cargo.toml

```diff
@@ -228,11 +233,16 @@ plugin = [
   "nu-protocol/plugin",
   "nu-engine/plugin",
 ]
+# Support for UNIX domain sockets / Windows named pipes
+# Currently only used for plugins
+local-socket = ["nu-parser/local-socket"]
```

Comment:
I am inclined to say that this may be premature configuration. Are there platforms that otherwise fail? 
While I can be talked into making this a feature gate for plugin authors, I think Nushell itself should be ready to work with the plugins you throw at it so it should not be another thing that may be missing.

Reply:
I think that's fair. I'm not aware of anywhere that it's an issue at the moment. I expect most platforms that we would consider supporting to have some kind of equivalent, and if they don't we can always just make it a runtime error and avoid doing it.

I agree with removing this feature, then. I actually originally added it because test support doesn't need it, but then I had issues with mismatched features and just let it keep it in there. So I think there's really no harm in undoing this.